### PR TITLE
 TIP-872: Move the last classes fron API bundle/component to the right BC

### PR DESCRIPTION
**Pull request by arnolanglade**
**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added legacy Behats               | -
| Added acceptance tests            | -
| Added integration tests           | -
| Changelog updated                 | yes
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | -
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: CHANGELOG-3.0.md [hunk 1]
```diff
@@ -388,4 +388,14 @@
...
+- Move `Akeneo\Tool\Bundle\ApiBundle\Controller\ProductModelController` to `Akeneo\Pim\Enrichment\Bundle\Controller\ExternalApi\ProductModelController`
+- Move `Akeneo\Tool\Bundle\ApiBundle\Doctrine\ORM\Repository\ProductRepository` to `Akeneo\Pim\Enrichment\Bundle\Doctrine\ORM\Repository\ExternalApi\ProductRepository`
+- Move `Akeneo\Tool\Component\Api\Normalizer\CategoryNormalizer` to `Akeneo\Pim\Enrichment\Component\Category\Normalizer\ExternalApi\CategoryNormalizer`
+- Move `Akeneo\Tool\Component\Api\Normalizer\ProductNormalizer` to `Akeneo\Pim\Enrichment\Component\Product\Normalizer\ExternalApi\ProductNormalizer`
+- Move `Akeneo\Tool\Component\Api\Repository\ProductRepositoryInterface` to `Akeneo\Pim\Enrichment\Component\Product\Repository\ExternalApi\ProductRepositoryInterface`
+- Move `Akeneo\Tool\Component\Api\Updater\ProductModelUpdater` to `Akeneo\Pim\Enrichment\Component\Product\ExternalApi\Updater\ProductModelUpdater`
+- Move `Akeneo\Tool\Component\Api\Updater\ProductModelUpdater` to `Akeneo\Pim\Enrichment\Component\Product\ExternalApi\Updater\ProductModelUpdater`
```
Comment (rouliane) [inline CHANGELOG-3.0.md]: This line is duplicated

File: UPGRADE.md [hunk 2]
```diff
@@ -354,20 +354,30 @@ find ./src/ -type f -print0 | xargs -0 sed -i 's/Pim\\Bundle\\EnrichBundle\\Form
...
+find ./src/ -type f -print0 | xargs -0 sed -i 's/Akeneo\\Tool\\Component\\Api\\Normalizer\\CategoryNormalizer/Akeneo\\Pim\\Enrichment\\Component\\Category\\Normalizer\\ExternalApi\\CategoryNormalizer/g'
+find ./src/ -type f -print0 | xargs -0 sed -i 's/Akeneo\\Tool\\Component\\Api\\Normalizer\\ProductNormalizer/Akeneo\\Pim\\Enrichment\\Component\\Product\\Normalizer\\ExternalApi\\ProductNormalizer/g'
+find ./src/ -type f -print0 | xargs -0 sed -i 's/Akeneo\\Tool\\Component\\Api\\Repository\\ProductRepositoryInterface/Akeneo\\Pim\\Enrichment\\Component\\Product\\Repository\\ExternalApi\\ProductRepositoryInterface/g'
+find ./src/ -type f -print0 | xargs -0 sed -i 's/Akeneo\\Tool\\Component\\Api\\Updater\\ProductModelUpdater/Akeneo\\Pim\\Enrichment\\Component\\Product\\ExternalApi\\Updater\\ProductModelUpdater/g'
+find ./src/ -type f -print0 | xargs -0 sed -i 's/Akeneo\\Tool\\Component\\Api\\Updater\\ProductModelUpdater/Akeneo\\Pim\\Enrichment\\Component\\Product\\ExternalApi\\Updater\\ProductModelUpdater/g'
```
Comment (AymericPlanche) [inline UPGRADE.md]: duplicated as the one in the changelog

File: UPGRADE.md [hunk 3]
```diff
@@ -354,20 +354,30 @@ find ./src/ -type f -print0 | xargs -0 sed -i 's/Pim\\Bundle\\EnrichBundle\\Form
...
-find ./src/ -type f -print0 | xargs -0 sed -i 's/Pim\Bundle\CommentBundle\Entity\Comment/Akeneo\Pim\Enrichment\Component\Comment\Model\Comment/g'
-find ./src/ -type f -print0 | xargs -0 sed -i 's/Pim\Bundle\CommentBundle\Model\CommentInterface/Akeneo\Pim\Enrichment\Component\Comment\Model\CommentInterface/g'
-find ./src/ -type f -print0 | xargs -0 sed -i 's/Pim\Bundle\CommentBundle\Model\CommentSubjectInterface/Akeneo\Pim\Enrichment\Component\Comment\Model\CommentSubjectInterface/g'
-find ./src/ -type f -print0 | xargs -0 sed -i 's/Pim\Bundle\CommentBundle\Normalizer\Standard\CommentNormalizer/Akeneo\Pim\Enrichment\Component\Comment\Normalizer\Standard\CommentNormalizer/g'
-find ./src/ -type f -print0 | xargs -0 sed -i 's/Pim\Bundle\CommentBundle\Repository\CommentRepositoryInterface/Akeneo\Pim\Enrichment\Component\Comment\Repository\CommentRepositoryInterface/g'
+find ./src/ -type f -print0 | xargs -0 sed -i 's/Pim\\Bundle\\PdfGeneratorBundle\\Controller\\ProductController/Akeneo\\Pim\\Enrichment\\Bundle\\Controller\\Ui\\ProductController/g'
```
Comment (jjanvier) [inline UPGRADE.md]: why does it remove and add stuff here?

Comment (arnolanglade) [reply UPGRADE.md]: `\` becomes `\\`

**Comment by arnolanglade** [comment]
Front tests does not worked for the last nightly building. 1 behat failed but it passes locally.

**Pull request merged**